src/ansys/fluent/core/session.py

```diff
@@ -269,13 +269,9 @@ def exit(self, **kwargs) -> None:
         self.fluent_connection.exit(**kwargs)
 
     def force_exit(self) -> None:
-        """Terminate session."""
+        """Terminate session (Both stand-alone and Docker container)."""
         self.fluent_connection.force_exit()
```

Comment:
Should this one have the same docstring as `fluent_connection.force_exit()` ? Or at least reference that one, which is more complete.